picker/crafter.lua

```diff
@@ -6,8 +6,8 @@ local lib = require("picker.lib")
 local function picker_crafter(event)
     local player = game.players[event.player_index]
     if player.selected then
-        local _, _, ip = lib.get_placeable_item(player.selected)
-        if ip and game.recipe_prototypes[ip.name] and player.force.recipes[ip.name].enabled and player.get_item_count(ip.name) == 0 then
+    		local _, _, ip = lib.get_placeable_item(player.selected)
```

Comment:
This will need a per-user setting, Some people combine Q and Alt-Q and this would needlessly spam new crafts